include/nbl/builtin/common.h

```diff
@@ -16,22 +16,9 @@
 
 namespace nbl::builtin
 {
```

Comment:
we can nuke this entire header now cause its autogenerated in `build`

Reply:
nuked in following commit